index.js

```diff
@@ -14,7 +14,10 @@ module.exports = {
 		worker: true,
 		es6: true, // Enables ES6 globals
 		'shared-node-browser': true, // restrict to common globals to preserve isomorphic support
-		commonjs: true
+		commonjs: true,
+		browser: true,
+		jest: true,
+		node: true
```

Comment:
What is the ordering rule inside `env` block?

Reply:
There were no ordering rules but if needs, alphabetical order seems appropriate.